src/main/java/no/sikt/nva/channelregister/ChannelRegister.java

```diff
@@ -172,6 +186,30 @@ public String lookUpInPublisher(String publisher) {
         }
     }
 
+    private String lookupInPublisherAliases(String publisher,
+                                            String customer) {
+        String pid = lookupInAliases(channelRegisterAliasesPublishers, publisher);
```

Comment:
var

Reply:
Er det en grunn til vi har eget oppslag av ntnu poster?

Reply:
changed to var